tests/test_config_parsing/summary_generator.py

```diff
@@ -231,15 +232,21 @@ class Unsmry:
     steps: List[SummaryStep]
 
     def to_ecl(self):
-        return [i for step in self.steps for i in step.to_ecl()]
+        a = [i for step in self.steps for i in step.to_ecl()]
+        return a
 
     def to_file(
         self, filelike, file_format: ecl_data_io.Format = ecl_data_io.Format.UNFORMATTED
     ):
         ecl_data_io.write(filelike, self.to_ecl(), file_format)
 
 
-positive_floats = st.floats(min_value=0.1, allow_nan=False, allow_infinity=False)
+positive_floats = from_dtype(
+    np.dtype(np.float32),
+    min_value=0.10000000149011612,  # 0.1 rounded to float32
```

Comment:
Hm, why not `min_value=0.1`?

Reply:
Can you put this line as a comment there ? Just that people / us would understand when we look at it in a year or so :smile:

Reply:
Comment updated! :)

Reply:
No, changed to this instead:
`
+    min_value=np.float32(0.1),
`